gpcontrib/arenadata_toolkit/sql/adb_collect_table_stats_test.sql

```diff
@@ -0,0 +1,58 @@
+CREATE EXTENSION arenadata_toolkit;
+SET search_path = arenadata_toolkit;
```

Comment:
Are there any collisions between the temporary table `db_files_current` from the `arenadata_toolkit.adb_collect_table_stats` function and the `arenadata_toolkit.db_files_current` table?